setup.py

```diff
@@ -125,7 +125,7 @@ def find_version(*parts):
         extra_compile_args = [
             '/EHsc',
             '-DBOOST_DATE_TIME_NO_LIB',
-            '/std:c++14',
+            '/std:c++1y',
```

Comment:
I don't think MSVC has c++1y implemented, so c++14 should be used. (or nothing)

Reply:
We only use/support gcc on windows, right? (I'm just following cmdstan and what seemed to work on httpstan)

Reply:
Yes. This part is a bit legacy and is here if MSVC starts to work at some point.